vllm/model_executor/layers/fused_moe/flashinfer_cutlass_prepare_finalize.py

```diff
@@ -3,9 +3,12 @@
 from typing import Optional
 
 import torch
+from flashinfer.comm.trtllm_alltoall import MnnvlMoe, MoEAlltoallInfo
```

Comment:
Maybe this import needs to be moved into `flashinfer_alltoall_dispatch` to avoid the test failures in CI?